### PR TITLE
modify logger, add logger_set level

### PR DESCRIPTION
**Pull request by Songyuanwei**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs
#429 
(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: tools/infer/text/parallel/predict_system.py [hunk 1]
```diff
@@ -12,8 +13,11 @@
 import cv2
 import numpy as np
 
+from mindocr.utils.logger import set_logger
 from mindocr.utils.visualize import recover_image
 
+logger = logging.getLogger("mindocr")
```
Comment (zhtmike) [inline tools/infer/text/parallel/predict_system.py:19]: 有些是 _logger， 有些是logger，保持一致会比较好

File: tools/modelarts_adapter/modelarts.py
Comment (zhtmike) [inline tools/modelarts_adapter/modelarts.py:1]: 不建议在modelarts.py 使用logger，因为modelart 会比 mindocr 先import，然后在modelart装必须的package等等再import mindocr。如果在这里预先import mindocr会出module missing等问题

File: tools/infer/text/parallel/predict_system.py (hunk 1 above)
Comment (Songyuanwei) [reply tools/infer/text/parallel/predict_system.py:19]: 主脚本用的是logger,例如train.py, eval.py等，其余调用的脚本用的_logger

File: tools/modelarts_adapter/modelarts.py
Comment (Songyuanwei) [reply tools/modelarts_adapter/modelarts.py:1]: 这里没有调用mindocr,之前调用的现在去掉了

**Review by zhtmike — APPROVED** [review]

File: mindocr/utils/callbacks.py [hunk 2]
```diff
@@ -152,7 +146,7 @@ def on_train_step_end(self, run_context):
                 f"loss: {loss:.6f}, " + lr_str + f"per step time: {per_step_time:.3f} ms, fps: {fps:.2f} img/s"
             )
```
Comment (zhtmike) [inline mindocr/utils/callbacks.py:148]: 这里要加个if main_rank 保护，否则distributed training message会重复出现

Comment (Songyuanwei) [reply mindocr/utils/callbacks.py:148]: 我试过了，分布式之后0卡打印，在logger.py文件设置了，只有0卡打印，然后main.py调用会全局生效。

File: tools/infer/text/parallel/utils_predict.py [hunk 3]
```diff
@@ -1,14 +1,13 @@
...
 import shutil
 
 import numpy as np
 import yaml
 from addict import Dict
 
-from mindocr.utils.logger import Logger
-
-_logger = Logger("mindocr")
+_logger = logging.getLogger("mindocr")
```
Comment (hadipash) [inline tools/infer/text/parallel/utils_predict.py:10]: `_logger = logging.getLogger(__name__)`?

File: tools/eval.py [hunk 4]
```diff
@@ -33,30 +36,26 @@ def main(cfg):
             parallel_mode="data_parallel",
             gradients_mean=True,
         )
+        set_logger(
+            name="mindocr", output_dir=cfg.train.ckpt_save_dir or "./", log_fn=f"log_eval_{rank_id}.txt", rank=rank_id
```
Comment (hadipash) [inline tools/eval.py:40]: The name of logger here (`mindocr`) is different form initialized above (`mindocr.eval`)

**Review by SamitHuang — APPROVED** [review]

**Pull request merged**